homeassistant/components/cast/media_player.py

```diff
@@ -1130,7 +1130,18 @@ def app_name(self):
     @property
     def supported_features(self):
         """Flag media player features that are supported."""
-        return SUPPORT_CAST
+        support = SUPPORT_CAST
+        media_status, _ = self._media_status()
+
+        if media_status:
+            if media_status.supports_queue_next:
+                support |= SUPPORT_PREVIOUS_TRACK
+            if media_status.supports_queue_next:
+                support |= SUPPORT_NEXT_TRACK
+            if media_status.supports_seek:
+                support |= SUPPORT_SEEK
```

Comment:
Setting this doesn't seem to make any difference to the frontend, is it not implemented?

Reply:
nope